apps/system/js/action_menu.js

```diff
@@ -107,7 +107,7 @@
     var target = evt.target;
     var defaultSelected = false;
     if (this.askForDefaultChoice) {
-      defaultSelected = !!this.defaultChoiceInput.getAttribute('checked');
+      defaultSelected = !!this.defaultChoiceInput.checked;
```

Comment:
Oh wait, `defaultChoiceInput` is a `<gaia-checkbox>`, not a native DOM element. You would need to make sure `<gaia-checkbox>` impl do that first.